pkg/test/resourcefixture/testdata/basic/bigqueryanalyticshub/v1alpha1/bigqueryanalyticshubdataexchange/bigqueryanalyticshubdataexchangebasic/create.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: bigqueryanalyticshub.cnrm.cloud.google.com/v1alpha1
 kind: BigQueryAnalyticsHubDataExchange
 metadata:
-  name: bigqueryanalyticshubdataexchange-${uniqueId}
+  name: bigqueryanalyticshubdataexchange${uniqueId}
```

Comment:
Out of interest, why the change?

Reply:
I've started using the name as the dataExchangeId and it only allows a-z, 0-9 characters IIRC